images/win/scripts/SoftwareReport/SoftwareReport.Databases.psm1

```diff
@@ -9,6 +9,7 @@ function Get-PostgreSQLMarkdown
         Version = $pgVersion
         UserName = $env:PGUSER
         Password = $env:PGPASSWORD
+        EnvironmentVariables = "PGBIN=$env:PGROOT; <br> PGDATA=$env:PGDATA; <br> PGROOT=$env:PGBIN"

```

Comment:
Is it expected that `PGROOT` contains bin?

`PGBIN=C:\Program Files\PostgreSQL\12; <br> PGDATA=C:\Program Files\PostgreSQL\12\data; <br> PGROOT=C:\Program Files\PostgreSQL\12\bin`

Reply:
Looks like we need to swap PGBIN and PGROOT

Reply:
Thank you, I have fixed this